openshift/fluent-bit.cm.yaml

```diff
@@ -106,6 +247,22 @@ parameters:
     displayName: Target Namespace
     required: true
   - name: LOGGING_HOST_NAME
-    description: The hostname of our FLuentd service running further down the monitoring pipeline
-    displayName: Logging host name
-    required: true
+    description: The hostname of our Fluentd service running further down the monitoring pipeline
+    displayName: Fluentd host name
+    required: false
+    default:
```

Comment:
When required is false, you don't need a blank default specified, unless you want that value to always be filled in with something.